lib/active_model_serializers/callbacks.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from
+# https://github.com/rails/rails/blob/7f18ea14c8/activejob/lib/active_job/callbacks.rb
+require 'active_support/callbacks'
+
+module ActiveModelSerializers
+  # = ActiveModelSerializers Callbacks
+  #
+  # ActiveModelSerializers provides hooks during the life cycle of serialization and
+  # allow you to trigger logic. Available callbacks are:
+  #
+  # * <tt>around_render</tt>
+  #
+  module Callbacks
```

Comment:
I ask about the public API because this callbacks is more code that we need to maintain, If the idea is to provide a public API I think we need to add to the docs, if this is to be used only internally I think we can make this more simple without the callback.